modules/nf-core/bcftools/merge/main.nf

```diff
@@ -8,13 +8,12 @@ process BCFTOOLS_MERGE {
         'biocontainers/bcftools:1.20--h8b25389_0' }"
 
     input:
-    tuple val(meta), path(vcfs), path(tbis)
-    tuple val(meta2), path(fasta)
-    tuple val(meta3), path(fai)
-    path(bed)
+    tuple val(meta), path(vcfs), path(tbis), path(bed)
+    tuple val(meta2), path(fasta), path(fai)
```

Comment:
```suggestion
    tuple val(meta2), path(fasta)
    tuple val(meta2), path(fai)
```

I'd say this is more the nf-core way

Reply:
Could you explain, why this is the case ?
In the previous it was even `tuple val(meta3), path(fai)` and I don't understand why we separate the fasta from it's index 🤔